idtoken/validate_test.go

```diff
@@ -64,7 +74,7 @@ func TestValidateRS256(t *testing.T) {
 					}
 				}),
 			}
-
+			now = tt.nowFunc
```

Comment:
Reset after the test? Don't want to accidentally mess up another test.

Reply:
Yes, done.